lib/node_modules/@stdlib/ndarray/base/mskfilter/lib/2d_accessors.js

```diff
@@ -0,0 +1,180 @@
+* var realf = require( '@stdlib/complex/realf' );
+* var imagf = require( '@stdlib/complex/imagf' );
+*
+* // Define the shape of the array:
+* var shape = [ 2, 2 ];
+*
+* // Define the array strides:
+* var sx = [ 2, 1 ];
+* var sm = [ 2, 1 ];
+* var sy = [ 1 ];
+*
+* // Define the index offset:
+* var ox = 0;
+* var om = 0;
+*
+* // Define getters and setters:
+* function getter( buf, idx ) {
+*     return buf.get( idx );
+* }
+*
+* function setter( buf, idx, value ) {
+*     buf.set( value, idx );
+* }
+*
+* var x = {
+*     'dtype': 'complex64',
+*     'data': new Complex64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0 ] ),
+*     'shape': shape,
+*     'strides': sx,
+*     'offset': ox,
+*     'order': 'row-major',
+*     'accessors': [ getter, setter ]
+* };
+*
+* var mask = {
+*     'dtype': 'bool',
+*     'data': new BooleanArray( [ 1, 0, 1, 0 ] ),
+*     'shape': shape,
+*     'strides': sm,
+*     'offset': om,
+*     'order': 'row-major',
+*     'accessors': [ getter, setter ]
+* };
+*
+* var y = {
+*     'dtype': 'complex64',
+*     'data': new Complex64Array( [ 0.0, 0.0, 0.0, 0.0 ] ),
+*     'shape': [ 2 ],
+*     'strides': sy,
+*     'offset': 0,
+*     'order': 'row-major',
+*     'accessors': [ getter, setter ]
+* };
+*
+* mskfilter2d( x, mask, y );
+*
+* var v = y.data.get( 0 );
+* // returns <Complex64>
+*
+* var re = realf( v );
+* // returns 1.0
+*
+* var im = imagf( v );
+* // returns 2.0
+*/
+function mskfilter2d( x, mask, y ) {
+	var xbuf;
+	var mbuf;
+	var ybuf;
+	var getx;
+	var getm;
+	var sety;
+	var dx0;
+	var dx1;
+	var dm0;
+	var dm1;
+	var dy0;
+	var S0;
+	var S1;
+	var sh;
+	var sm;
+	var sx;
+	var ix;
+	var im;
+	var iy;
+	var i0;
+	var i1;
+
+	// Extract loop variables: dimensions and loop offset (pointer) increments...
+	sh = x.shape;
+	sx = x.strides;
+	sm = mask.strides;
+	if ( x.order === 'row-major' ) {
+		// For row-major ndarrays, the last dimensions have the fastest changing indices...
+		S0 = sh[ 1 ];
+		S1 = sh[ 0 ];
+		dx0 = sx[ 1 ];                // offset increment for innermost loop
+		dx1 = sx[ 0 ] - ( S0*sx[1] ); // offset increment for outermost loop
+		dm0 = sm[ 1 ];
+		dm1 = sm[ 0 ] - ( S0*sm[1] );
+	} else { // order === 'column-major'
+		// For column-major ndarrays, the first dimensions have the fastest changing indices...
+		S0 = sh[ 0 ];
+		S1 = sh[ 1 ];
+		dx0 = sx[ 0 ];                // offset increment for innermost loop
+		dx1 = sx[ 1 ] - ( S0*sx[0] ); // offset increment for outermost loop
+		dm0 = sm[ 0 ];
+		dm1 = sm[ 1 ] - ( S0*sm[0] );
+	}
+	dy0 = y.strides[ 0 ];
+
+	// Set the pointers to the first indexed elements in the respective ndarrays...
+	ix = x.offset;
+	im = mask.offset;
+	iy = y.offset;
+
+	// Cache references to the input and output ndarray buffers...
+	xbuf = x.data;
+	mbuf = mask.data;
+	ybuf = y.data;
+
+	// Cache accessors:
+	getx = x.accessors[ 0 ];
+	getm = mask.accessors[ 0 ];
+	sety = y.accessors[ 1 ];
```

Comment:
```suggestion
	xget = x.accessors[ 0 ];
	mget = mask.accessors[ 0 ];
	yset = y.accessors[ 1 ];
```

Use consistent naming convention with `xbuf` et al.

Reply:
This applies here and elsewhere.